integration-tests/kotlin-serialization/src/main/kotlin/io/quarkus/it/kotser/GreetingApplication.kt

```diff
@@ -0,0 +1,11 @@
+package io.quarkus.it.kotser
+
+import io.quarkus.runtime.Quarkus.run
+import io.quarkus.runtime.annotations.QuarkusMain
+
+@QuarkusMain
```

Comment:
Create a case like this:

```kotlin
@QuarkusMain
class GreetingApplication {
   fun main(args: Array<String>) {
       run(*args)
   }
}
```
and see how things fare.  I have hunch it will break.